### ISSUE TITLE
YQL query never stops

### ISSUE THREAD

**Issue by kaikash** [ytsaurus/ytsaurus#247]
There is table with column of ~80KiB for each row. And there is query
```SQL
select * from table limit 1000
```

This query never stops and looping with error:
```
2023-12-13 12:20:06,555388	E	QueryHandler	Failed to write query state, backing off (QueryId: 32bb7be8-27058e7b-9c9b6957-2e67a7a3, Engine: Yql)
Error committing transaction 195e5dd1-8000002d-60002-e81bd78d
    origin          1.query-tracker.cluster.local (pid 233, thread Connection:1, fid fffef2828492ae3f)
    datetime        2023-12-13T09:20:06.552442Z
    cluster_id      Native(Name=jupiter)

  Error preparing rows for table //sys/query_tracker/finished_query_results
      origin          1.query-tracker.cluster.local (pid 233, thread Connection:1, fid fffef2828492ae3f)
      datetime        2023-12-13T09:20:06.552396Z

    Value of type "string" is too long for dynamic data: length 81346952, limit 16777216
        code            326
        origin          1.query-tracker.cluster.local (pid 233, thread Connection:1, fid fffef2828492ae3f)
        datetime        2023-12-13T09:20:06.552354Z
	Control	fffef2890fdc66c9	ed6df440-fee035d9-744836fd-333048c9
```

**Comment by kaikash** [comment]
Any updates?

**Comment by Krisha11** [comment]
The error has not been fixed at the moment. Someone will fix this at some point in the foreseeable future.

This error occurs because queries with large results are processed incorrectly

As a temporary solution, you can avoid queries that return large responses. For example, in this situation, you can reduce the limit to about 200 lines.

To remove a problematic query from the history, you can delete the corresponding row from the system tables in `//sys/query_tracker`

**Comment by Krock21** [comment]
Fixed in [Query Tracker 0.0.6](https://github.com/ytsaurus/ytsaurus/releases/tag/docker%2Fquery-tracker%2F0.0.6)
